docs/release-strategy.md

```diff
@@ -29,10 +29,16 @@ PRs and Issues associated with the next two milestones will be prioritized for r
 
 ## Git Branches and Tags
 
-Every `X.Y` release stream gets a new branch.
+Every `X.Y` release stream gets a new branch `release-X.Y`. The release branch is created with first beta, release candidate, or final `X.Y.0` release.
 
 Each release, `X.Y.Z`, exists as a tag named `vX.Y.Z`.
 
+Alpha releases are tagged as `vX.Y.Za1`, `vX.Y.Za2`, and so on. Alpha releases are created from the main branch, **not** from a `release-X.Y` branch.
```

Comment:
```suggestion
Alpha releases are tagged as `vX.Y.Zalpha1`, `vX.Y.Za2`, and so on. Alpha releases are created from the main branch, **not** from a `release-X.Y` branch.
```

Going with fullname is clearer, it's cheap and users will immediately understand.

Reply:
> I'm sorry, Dave. I'm afraid I can't do that.
> HAL 9000

We have to follow https://peps.python.org/pep-0440/#pre-releases

Reply:
🤷🏻‍♂️